webapp/store/views.py

```diff
@@ -270,15 +277,22 @@ def store_category(category):
 
         try:
             category_results = api.get_searched_snaps(
-                snap_searched="", category=category, size=24, page=1
+                snap_searched="", category=category, size=10, page=1
             )
         except ApiError as api_error:
             status_code, error_info = _handle_errors(api_error)
 
         snaps_results = logic.get_searched_snaps(category_results)
 
+        if len(snaps_results) > 0 and snaps_results[0]["icon_url"] == "":
+            snaps_results = snaps_results[:-1]
+
+        for snap in snaps_results:
+            snap["show_summary"] = True
```

Comment:
If all snaps are getting "show_summary" why do we need it at all? Can't it be treated as True by default?